### ISSUE TITLE
Navigation and interaction should be usable with a multi-frequency, binary input system

### ISSUE THREAD

**Issue by misslivirose** [mozilla/hubs#1883]
**Is your feature request related to a problem? Please describe.**
While Hubs provides a number of interaction paradigms dependent on device type; we should have a mode that we can distill down core interactions and navigation to a binary input control with varying input frequencies (short/long/continuous). 

I started thinking about this as a result of this work: 
https://blog.prototypr.io/accessible-locomotion-and-interaction-in-webxr-e4d87c512e51

More detail to come.

**Comment by misslivirose** [comment]
Consider as part of longer-term sprint on accessibility
